tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -0,0 +1,170 @@
+// EnergyPlus::SimAirServingZones Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// C++ Headers
+#include <cassert>
+#include <cmath>
+#include <string>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+#include <ObjexxFCL/Fmath.hh>
+#include <ObjexxFCL/gio.hh>
+
+// EnergyPlus Headers
+#include <SimAirServingZones.hh>
+#include <BranchInputManager.hh>
+#include <DataAirLoop.hh>
+#include <DataAirSystems.hh>
+#include <DataContaminantBalance.hh>
+#include <DataConvergParams.hh>
+#include <DataDefineEquip.hh>
+#include <DataEnvironment.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalance.hh>
+#include <DataHVACControllers.hh>
+#include <DataHVACGlobals.hh>
+#include <DataLoopNode.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataSizing.hh>
+#include <DataSystemVariables.hh>
+#include <DataZoneEquipment.hh>
+#include <DesiccantDehumidifiers.hh>
+#include <EMSManager.hh>
```

Comment:
Maybe I can just block the headers that I don't use. Then other developers can just unblock them if they need them. It will be less time-consuming for them.